Sources/SwiftSyntaxMacros/MacroSystem.swift

```diff
@@ -394,6 +403,43 @@ extension MacroApplication {
     return peers
   }
 
+  // If any of the custom attributes associated with the given declaration
+  // refer to conformance macros, expand them and return the resulting
+  // set of extension declarations.
+  private func expandConformances(of decl: DeclGroupSyntax) -> [DeclSyntax] {
+    let identifier: String
```

Comment:
Why not make `identifier: Syntax` and then you don’t need to convert to strings or  use the `raw` interpolation below. Also `identifier` isn’t correct because `ext.extendedType` is not an identifier.

Reply:
Great point, I've updated this code.